.github/docker-compose.test.yml

```diff
@@ -14,11 +14,35 @@
 services:
   # The Unleash server contains the Unleash configuration and
   # communicates with server-side SDKs and the Unleash Proxy
+  frontend:
+    image: node:20-alpine
+    working_dir: /app
+    volumes:
+      - ..:/src
+    command: |
+      sh -c " cp -r /src/* /app && \
```

Comment:
Because this is mounted as a volume, installing dependencies in this node-20-alpine image results in different binaries being installed than those for ubuntu-latest. This copy makes sure we don't touch the source files when building